src/main/java/apoc/refactor/util/RefactorConfig.java

```diff
@@ -32,7 +35,9 @@ public RefactorConfig(Map<String,Object> config) {
 			this.propertiesManagement = (Map<String,String>)value;
 		}
 
-		this.mergeRelsAllowed = config.get("mergeRels");
+		this.mergeRelsAllowed = toBoolean(config.get("mergeRels"));
+		this.selfRel = toBoolean(config.get("selfRel"));
+		this.countProperties = toBoolean(config.get("countProperties"));
```

Comment:
this was meant to mean "a property that holds the node/relationship count that was merged"